tests/evhtests/l7_evh_ingressclass_test.go

```diff
@@ -400,20 +439,19 @@ func TestEVHUpdateInfraSettingInIngressClass(t *testing.T) {
 	// update infrasetting ref in ingressclass, model changes
 	g := gomega.NewGomegaWithT(t)
 
-	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting", "my-infrasetting2"
+	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting1", "my-infrasetting2"
```

Comment:
We have changed infrasetting name here. Are we still facing timing related issue here ?

Reply:
No failures with the infrasetting here. My thought was to follow the same naming convention as `my-infrasetting2`.